src/encoded/schemas/analysis_step.json

```diff
@@ -342,6 +343,7 @@
                     "fine-mapped variants",
                     "fold change over control",
                     "footprints",
+                    "functional conservation scores",
```

Comment:
can we change it to "functional conservation quantifications"?

Reply:
Changed